core/chaincode/platforms/golang/list.go

```diff
@@ -92,6 +80,9 @@ func gopathDependencyPackageInfo(goos, goarch, pkg string) ([]PackageInfo, error
 		if packageInfo.Incomplete {
 			return nil, fmt.Errorf("failed to calculate dependencies: incomplete package: %s", packageInfo.ImportPath)
 		}
+		if packageInfo.Goroot {
```

Comment:
why not `Standard`? (and i wonder how it worked before w/o this check...)

Reply:
Looks at the old template, please. This is doing exactly what we had before - excluding packages that are from the go root - which includes more than “standard”.